src/connector/src/sink/catalog/desc.rs

```diff
@@ -61,7 +61,7 @@ impl SinkDesc {
         schema_id: SchemaId,
         database_id: DatabaseId,
         owner: UserId,
-        dependent_relations: Vec<TableId>,
+        dependent_relations: Vec<u32>,
```

Comment:
What's the semantics of `u32` here? I guess we're de facto using `TableId` as the general type for identifying all relations. 🥵

Reply:
The semantic is relation. 🥵 yes we need to introduce a general type for identifying all relations and generate ids for all of them from the same category in meta also(currently they all use `Table`).

Reply:
Reverted. Will do some refactor on the ids.